README.md

```diff
@@ -81,10 +81,16 @@ Unzip and copy the .framework to `AmazonChimeSDK` folder
 
 Deploy the serverless demo from [amazon-chime-sdk-js](https://github.com/aws/amazon-chime-sdk-js)
 
+This will give you two urls `https://xxxxx.xxxxx.xxx.com/Prod/` and `https://xxxxx.xxxxx.xxx.com/Prod/v2/` 
+
+We will provide `https://xxxxx.xxxxx.xxx.com/Prod/` for mobile demo app.
+
```

Comment:
Maybe save the subject ("We", "you") like other sentences?

"Deploy the serverless demo from [amazon-chime-sdk-js](https://github.com/aws/amazon-chime-sdk-js), which returns two urls `https://xxxxx.xxxxx.xxx.com/Prod/` and `https://xxxxx.xxxxx.xxx.com/Prod/v2/` 

Provide `https://xxxxx.xxxxx.xxx.com/Prod/` for mobile demo app."

Reply:
updated